### DISCUSSION TITLE
windows - How do you want to open this file?

### DISCUSSION THREAD

**Discussion by Schmell** [pocketbase/pocketbase#1894] [Q&A]
when i try `./pocketbase serve` i get the windows _**How do you want to open this file?**_ dialog.
it did work the first time i ran the command, but now i can't figure out what to do.
What do i select to run pocketbase ... java?

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean. You don't need java or any other dependency to run PocketBase. The shipped executable includes everything in itself and it is statically linked.

If you want to start the executable on Windows:
1. Download the archive for your platform:
   - for x86_x64: https://github.com/pocketbase/pocketbase/releases/download/v0.12.3/pocketbase_0.12.3_windows_amd64.zip
   - for arm64: https://github.com/pocketbase/pocketbase/releases/download/v0.12.3/pocketbase_0.12.3_windows_arm64.zip

2. Extract the archive

3. Open cmd or other terminal and run `\path\to\pocketbase.exe serve`

**Reply by Schmell** [reply]
Im not sure why Poweshell is not opening it i have to run it in cmd

**Comment by DuncanCooper** [comment]
I had the exact same experience and found that I had downloaded the `linux` executable of pocketbase instead of the `windows` executable. 

The linux version is called `pocketbase` while the windows version is called `pocketbase.exe` so you may done the same. 

With the correct version in place, `pocketbase.exe serve` worked for me.
